### PR TITLE
Feat/st 844: Code updated just for testing in AAT environment

### PR DESCRIPTION
**Pull request by sjamihmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ST-844

### Change description ###
 Code updated just for testing in AAT env as we are not able to test this in local env

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by muradalamv1 — APPROVED** [review]

**Comment by muradalamv1** [comment]
There are commented code , can you remove please

File: src/main/java/uk/gov/hmcts/sptribs/common/notification/DecisionIssuedNotification.java [hunk 1]
```diff
@@ -121,14 +120,21 @@ private NotificationResponse sendLetterNotification(Map<String, Object> template
...
         final String authorisation = idamService.retrieveSystemUpdateUserDetails().getAuthToken();
         String serviceAuthorization = authTokenGenerator.generate();
 
-        if (caseIssueDecision.getDecisionDocument() != null) {
+        CicCase cicCase = caseData.getCicCase();
+        /*if (caseIssueDecision.getDecisionDocument() != null) {
             List<String> uploadedDocumentsUrls = caseIssueDecision.getDecisionDocument().stream().map(item -> item.getValue())
+                .map(item -> StringUtils.substringAfterLast(item.getDocumentLink().getUrl(), "/"))
+                .collect(Collectors.toList());*/
+
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/common/notification/DecisionIssuedNotification.java:134]: Commented out code?

**Review by osheaf01 — CHANGES_REQUESTED** [review]
Remove the commented out code - unless there's a really good reason to keep it in?

File: src/main/java/uk/gov/hmcts/sptribs/common/notification/DecisionIssuedNotification.java (hunk 1 above)
Comment (sjamihmcts) [reply src/main/java/uk/gov/hmcts/sptribs/common/notification/DecisionIssuedNotification.java:134]: Fergal, this code is temporary, I need to test using already saved document as per TODO comment. Will revert this code once i test this scenario in AAT env as i cant do in local env.

**Review by osheaf01 — APPROVED** [review]
Approved

**Pull request merged**